### ISSUE TITLE
timeSeriesListMerge produce undefined value

### ISSUE THREAD

**Issue by renhax** [esnet/pond#128]
Hi,
I'm having issue when to merge two time series with different time range, the column has less time range will have some undefined value. It will cause some issue when I draw the graph, is there any way to fill those undefined value to null or 'N/A', or remove after merge completely. I notice fill doesn't support that.

Thanks,
Renhao

**Comment by gingerizer** [comment]
I've also observed this and attempted to use timeSeries.fill(options) in an attempt to replace the undefined values with zero.   The fill doesn't seem to work however, likely because undefined is not currently considered missing or invalid.   Perhaps I'm using fill incorrectly?  In either case, the documentation on the fill function isn't clear on the definition of missing or invalid, or on how to replace undefined values with zeros.    

I'm currently forced to manually correct the point array, and create a new TimeSeries which is less than ideal.

**Comment by pjm17971** [comment]
Can either of you post some code that shows the problem and I can look into what the best solution is, or add a enhancement request to cove that use case? Also, what version of Pond are you using?

Currently fill isn't going to help you, it's for actual missing values along a given period. i.e. the value actually has to be missing. It might be a fairly easy change to optionally allow fill to support null/undefined/NaN type invalid values as missing.

**Comment by gingerizer** [comment]
Created this example quickly using data from react-timeseries-charts examples.   As I was creating this I realized I forgot to mention that this happens in the case when the series being merged together don't have common column names so they can't be concat'd together by the merge.      Obviously in the example I've created it doesn't make sense to change the column names but in my real world case it makes sense.  I have distinct series that I want to display on a stacked bar chart which requires a single series with independent columns.   

In the [example](https://codesandbox.io/s/nwkw9qxr44), the second time series is missing the first 5 points from the original series.   The first 5 points in the array contain undefined values for the cropped series column.  

I.e. merged time series 
```
mergedSeries = { 
   name: "MergedSeries", 
   utc:  true
   columns:  [index, percip, percip_cropped]
   points:  [ ["1h-412565", 0.01, undefined] , 
            ...]
  }
```

**Comment by gingerizer** [comment]
Yes, it would be nice if fill would treat undefined as a missing vlaue.    Just for reference, to fix the merged series points I did the following using lodash:
 
```
const fixedSeriesPoints = mergedSeries.toJSON().points.map(point => {
   return point.map(p => {
      return _.isUndefined(p) ? 0 : p;
   });
});

const fixedSeries = new TimeSeries({
   name: "MergedSeries",
   columns: mergedSeries.toJSON().columns,
   points: fixedSeriesPoints 
});
```
